### PR TITLE
adding pytest-harvest recipe

### PR DESCRIPTION
**Pull request by harelwa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pytest-harvest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
```
Comment (beenje) [inline recipes/pytest-harvest/meta.yaml:20]: This shouldn't be needed to build the package.

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/pytest-harvest/meta.yaml (hunk 1 above)
Comment (harelwa) [reply recipes/pytest-harvest/meta.yaml]: you are right. removed. thank you.

File: recipes/pytest-harvest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 532f99080aa85e8bfc7cb2ae572afaa77df727c4fc3a8f319f11669320cd4d24
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
```
Comment (beenje) [inline recipes/pytest-harvest/meta.yaml:19]: Any reason you changed the python min version here?
I think the same min version should be used in host and run.

Comment (harelwa) [reply recipes/pytest-harvest/meta.yaml:19]: yes, I wanted `>=` of EOL py versions. any objections ?

https://endoflife.date/python

aligned `run` requirement as well.

**Pull request merged**